examples/external_arguments.rs

```diff
@@ -0,0 +1,38 @@
+extern crate fluent;
+
+use fluent::context::MessageContext;
+use fluent::context::FluentArgument;
+use std::collections::HashMap;
+
+fn main() {
+    let locales = vec![String::from("pl")];
+    let mut ctx = MessageContext::new(locales);
+
+    ctx.add_messages("hello-world = Hello { $name }");
+    ctx.add_messages("ref = The previous message says { hello-world }");
+    ctx.add_messages("unread-emails = You have { $emailCount } unread emails");
+
+    let mut args = HashMap::new();
+    args.insert(String::from("name"), FluentArgument::from("John"));
```

Comment:
Is there a way to use `&str` as keys of the `args` map? I'm not sure how `args.get` would then look them up, but having to use `String::from` looks a bit unwieldy.